docs/faq.md

```diff
@@ -186,6 +189,6 @@ A policy is a simple collection of rules and a definition of what to do with rul
 
 ### Do you have a visualization UI?
```

Comment:
```suggestion
### Do you have a graphical user interface?
```

Reply:
@s-santillan I know either user interface (UI) or graphical user interface (GUI). I have not seen a word "visualization UI" before, as I think it comes from "data visualization". See https://stackoverflow.com/questions/3191844/what-is-the-difference-between-gui-and-ui - we should probably use "user interface" looking at the context of this FAQ, what do you think?

```suggestion
### Do you have a user interface?
```

Reply:
@s-santillan I was just trying to point out that this is not a standard vocabulary.